terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -870,6 +870,7 @@ module "slmicro60-minion" {
 // WORKAROUND: Does not work in sumaform, yet
 //  additional_packages = [ "venv-salt-minion" ]
 //  install_salt_bundle = true
+  install_salt_bundle = false
```

Comment:
```suggestion
  install_salt_bundle = false
```

Reply:
For now, I prefer to keep the whole block commented:
```
// WORKAROUND: Does not work in sumaform, yet
//  additional_packages = [ "venv-salt-minion" ]
//  install_salt_bundle = true
```

Reply:
OK, fine with me.